content/en/docs/collector/trace-receiver.md

```diff
@@ -71,16 +86,19 @@ docker run -d --name jaeger \
   jaegertracing/all-in-one:1.41
 ```
 
-Now, create a `config.yaml` file so you can set up your Collector's components.
+Once the Docker container is up and running, you can access Jaeger UI by:
+<http://localhost:16686/>
+
+Now, create a Collector config file named `config.yaml` to set up the
+Collector's components and pipelines.
 
 ```sh
-cd otelcol-dev
 touch config.yaml
 ```
 
 For now, you just need a basic traces pipeline with the `otlp` receiver, the
-`otlp` and `debug`[^1] exporters, here is what your `config.yaml` file should
-look like:
+`otlp` and `debug`[^1] exporters, and optionally the `batch` processor. Here is
```

Comment:
Why is `batch` included here? I would keep only the things that are relevant for the tutorial. If the idea is to have a recommended configuration in place, then it's missing the memory limiter, pprof, and possibly others.

Reply:
As the "Building a custom collector" doc is kind of the starting point, I respect the original doc [here](https://opentelemetry.io/docs/collector/custom-collector/), where the `batch` processor is included.

That's why I wanted to make them aligned but if you wanted to remove such an "optional" component, I can remove it and may send another PR to update that "Building a custom collector" doc as well.